tutorial/doppleganger.js

```diff
@@ -0,0 +1,529 @@
+"use strict";
+//======================================
+// Version 1.1
+// Addaption to "Local" Entities (since Overlays get deprecated for "Model" type.)
+// by Alezia Kurdis on February 2020
```

Comment:
```suggestion
// by Alezia Kurdis on February 2022
```
Is this a typo, or did you actually write this back in 2020?